### PR TITLE
APIGOV-16633 - update default traceability Condor endpoint/port

### PR DESCRIPTION
**Pull request by kflan-io**
New condor endpoint is live. You can verify:
curl ingestion.datasearch.axway.com:5044  returns "curl: (52) Empty reply from server"

### CODE REVIEW

**Review by sbolosan — APPROVED** [review]

File: docs/traceability/index.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ apic_traceability_agent:
 output.traceability:
   enabled: true
   hosts:
-   - ${TRACEABILITY_HOST:"ingestion-lumberjack.datasearch.axway.com:453"}
+   - ${TRACEABILITY_HOST:"ingestion.datasearch.axway.com:5044"}
```
Comment (jcollins-axway) [inline docs/traceability/index.md:265]: is this changing for EU as well?

**Review by jcollins-axway — APPROVED** [review]

File: docs/traceability/index.md (hunk 1 above)
Comment (vivekschauhan) [inline docs/traceability/index.md:265]: I hope we did a test with publishing events to ingestion.datasearch.axway.com:5044

**Review by Atraya21 — APPROVED** [review]

File: docs/traceability/index.md (hunk 1 above)
Comment (kflan-io) [reply docs/traceability/index.md:265]: I was able to view published events in a production env.

Comment (kflan-io) [reply docs/traceability/index.md:265]: Yes. 
ingestion.visibility.eu-fr.axway.com:5044

Comment (jcollins-axway) [reply docs/traceability/index.md:265]: Then we need to update the cli for this as well

Comment (vivekschauhan) [reply docs/traceability/index.md:265]: Perfect, thanks :)

**Review by vivekschauhan — APPROVED** [review]

File: docs/traceability/index.md (hunk 1 above)
Comment (tsjohns9) [reply docs/traceability/index.md:265]: Do we need this url in the cli as well?

Comment (kflan-io) [reply docs/traceability/index.md:265]: I created a sub-task for CLI changes. I will address those Monday and touch base w you.

**Pull request merged**